README.md

```diff
@@ -183,6 +183,14 @@ You can define new types with `struct`, `enum`, and `custom`. And you can constr
 [custom] allows you to send any C# type you wish through Slice, in keeping with IceRPC's mantra of modularity and
 extensibility. You just need to provide methods to encode and decode instances of your custom type.
 
+## Protobuf
+
+Protocol Buffers or Protobuf for short is a popular IDL and serialization format developed by Google. It's the preferred
```

Comment:
Should be some kind of separator for this parenthetical statement.
For some unexplainable reason, commas feel most correct to me.
```suggestion
Protocol Buffers, or Protobuf for short, is a popular IDL and serialization format developed by Google. It's the preferred
```